app/views/events/index.html.erb

```diff
@@ -15,8 +15,8 @@
         <td class="govuk-table__cell" scope="row"><%= event.type %></th>
         <td class="govuk-table__cell" scope="row"><code><%= JSON.pretty_generate(event.data) %></code></td>
         <td class="govuk-table__cell" scope="row"><%= event.created_at %></td>
-        <td class="govuk-table__cell" scope="row"><%= event.aggregate_id %></td>
-        <td class="govuk-table__cell" scope="row"><%= event.aggregate_type %></td>
+        <td class="govuk-table__cell" scope="row"><%= event&.aggregate_id %></td>
```

Comment:
How come we've changed to safe navigation operation here? Isn't the safe navigation operator only needed when you're chaining methods? like `foo&.bar&.baz`?

My understanding is that when there isn't an aggregate object that `event.aggregate_id` and `event&.aggregate_id` is equal. Is it related to the association on `AggregatedEvent`?

Reply:
You are probably right I did this with this thinking that if event was nil then nil.aggregate_id won't exist but nil&.aggregate_id is nil